integrator-broker-analytics/README.md

```diff
@@ -0,0 +1,229 @@
+# Kubernetes Resources for deployment of Integrator and Broker profiles of WSO2 Enterprise Integrator with Analytics
+
+Core Kubernetes resources for a clustered deployment of Integrator and Broker profiles of WSO2 Enterprise Integrator with Integrator Analytics support.
+
+![A clustered deployment WSO2 Enterprise Integator's Integrator and Broker profiles with Integrator Analytics support](integrator-broker-analytics.png)
+
+## Prerequisites
+
+* In order to use these Kubernetes resources, you will need an active [Free Trial Subscription](https://wso2.com/free-trial-subscription)
+from WSO2 since the referring Docker images hosted at docker.wso2.com contains the latest updates and fixes for WSO2 Enterprise Integrator.
+You can sign up for a Free Trial Subscription [here](https://wso2.com/free-trial-subscription).<br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Docker](https://www.docker.com/get-docker)
+(version 17.09.0 or above) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+in order to run the steps provided<br>in the following quick start guide.<br><br>
+
+* An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup/pick-right-solution/)<br><br>
+ 
+## Quick Start Guide
+
+>In the context of this document, `KUBERNETES_HOME` will refer to a local copy of the [`wso2/kubernetes-ei`](https://github.com/wso2/kubernetes-ei/)
+Git repository.<br>
+
+##### 1. Checkout Kubernetes Resources for WSO2 Enterprise Integrator Git repository:
+
+```
+git clone https://github.com/wso2/kubernetes-ei.git
+```
+
+##### 2. Create a namespace named `wso2` and a service account named `wso2svc-account`, within the namespace `wso2`.
+
+```
+kubectl create namespace wso2
+kubectl create serviceaccount wso2svc-account -n wso2
+```
+
+Then, switch the context to new `wso2` namespace from `default` namespace.
+
+```
+kubectl config set-context $(kubectl config current-context) --namespace=wso2
+```
+
+##### 3. Create a Kubernetes Secret for pulling the required Docker images from [`WSO2 Docker Registry`](https://docker.wso2.com):
+
+Create a Kubernetes Secret named `wso2creds` in the cluster to authenticate with the WSO2 Docker Registry, to pull the required images.
+
+```
+kubectl create secret docker-registry wso2creds --docker-server=docker.wso2.com --docker-username=<username> --docker-password=<password> --docker-email=<email>
+```
+
+`username`: Username of your Free Trial Subscription<br>
+`password`: Password of your Free Trial Subscription<br>
+`email`: Docker email
```

Comment:
What do we mean by `Docker email`? That's confusing, FT username would be an email right, can't we use the same here? Is the `--docker-email` used for authentication?

Reply:
+1 we can call it the FT username. It is used as part of the credentials that are passed through the Kubernetes Secret for pulling WSO2 Docker images from the Docker Private Registry [1].

[1]: [Kubernetes Secret](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-secret-in-the-cluster-that-holds-your-authorization-token)